### ISSUE TITLE
es6 import support.

### ISSUE THREAD

**Issue by clibu** [hildjj/node-inspect-extracted#8]
At present to use this module with es6 we need to use:
```
import pkg from 'node-inspect-extracted';
const { inspect } = pkg;
````
instead of:
````
import { inspect } from 'node-inspect-extracted';
````

**Comment by hildjj** [comment]
Hm.  I think to do this right, I'm going to have to turn the top level into an es6 module.  That's not the end of the world, since this should almost-always be used from the web.

**Comment by hildjj** [comment]
@clibu can you look at #9 please, and see if that will work for your use case?

**Comment by clibu** [comment]
@hildjj Sure can, it won't be until later today _(Australian time)._

**Comment by clibu** [comment]
@hildjj Ok I updated to v1.0.7 and still get: 
````
SyntaxError: Named export 'inspect' not found. The requested module 'node-inspect-extracted' is a CommonJS module, which may not support all module.exports as named exports.
````
using:
```import { inspect } from 'node-inspect-extracted';```

**Comment by hildjj** [comment]
I haven't merged that PR or done a release yet, so 1.0.7 is unchanged.  If you can clone the `wrapper` branch, you could try it out before I merge.  Otherwise, I'll think about how else to test this, and do a release in a day or two.
